packages/react-chat/package.json

```diff
@@ -42,12 +42,12 @@
     "test:unit": "yarn g:vitest run --coverage"
   },
   "dependencies": {
-    "@voiceflow/base-types": "2.97.3",
+    "@voiceflow/base-types": "2.111.4",
     "@voiceflow/dtos-interact": "1.1.0",
-    "@voiceflow/sdk-runtime": "1.7.0",
+    "@voiceflow/sdk-runtime": "workspace:*",
```

Comment:
`sdk-runtime` was part of a monorepo that we had archived so relocated it here since we still depend on it